pmm-app/tests/pages/dashboardPage.js

```diff
@@ -542,7 +542,10 @@ module.exports = {
     annotationMarker: '(//div[contains(@class,"events_marker")])',
     clearSelection: '//a[@ng-click="vm.clearSelections()"]',
     Last12HoursValue: "//span[contains(text(), 'Last 12 hours')]",
-    Last2Days: "//span[contains(text(), 'Last 2 days')]"
+    Last2Days: "//span[contains(text(), 'Last 2 days')]",
+    timeRangePickerButton: '.btn.navbar-button.navbar-button--tight',
```

Comment:
I think I forgot, but we have all related stuff to time range locators in adminPage, so probably would make sense to move that stuff out, no point in having same locators on two different pages, https://github.com/percona/grafana-dashboards/blob/PMM-2.0/pmm-app/tests/pages/adminPage.js#L54